util/cron/test-darwin.bash

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
 #
-# Test examples for default configuration on darwin
+# Test full suite for default configuration on darwin
 
 UTIL_CRON_DIR=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
 source $UTIL_CRON_DIR/common.bash
 source $UTIL_CRON_DIR/common-darwin.bash
+source $UTIL_CRON_DIR/common-localnode-paratest.bash
 
 export CHPL_NIGHTLY_TEST_CONFIG_NAME="darwin"
 
-$UTIL_CRON_DIR/nightly -cron -examples -blog
```

Comment:
will this stop testing the blog on darwin?

Reply:
Yes, which was not intended, will fix